dotfiles/prompt

```diff
@@ -160,3 +160,6 @@ _prompt() {
 }
 
 PROMPT_COMMAND='_prompt'
+if [[ "$TERM_PROGRAM" = Apple_Terminal ]]; then
+  export PROMPT_COMMAND="$PROMPT_COMMAND;update_terminal_cwd;"
```

Comment:
I think it's built in actually, just only available in Apple Terminal and not iTerm.

> On Jun 5, 2015, at 2:54 PM, Chris Wilson <notifications@github.com> wrote:
> 
> In dotfiles/prompt:
> 
> > @@ -160,3 +160,6 @@ _prompt() {
> >  }
> >  
> >  PROMPT_COMMAND='_prompt'
> > +if [[ "$TERM_PROGRAM" = Apple_Terminal ]]; then
> > +  export PROMPT_COMMAND="$PROMPT_COMMAND;update_terminal_cwd;"
> Is this missing the update_terminal_cwd command? Perhaps in another commit?
> 
> —
> Reply to this email directly or view it on GitHub.
>